docker-compose.yml

```diff
@@ -454,3 +456,35 @@ services:
       - SSL_CERTFILE=/tls.cert
       - SSL_KEYFILE=/tls.key
     command: ["Fulcrum", "/fulcrum.conf"]
+
+  svix:
+    image: svix/svix-server
+    platform: linux/amd64
+    environment:
+      WAIT_FOR: "true" # We want to wait for the default services
```

Comment:
For this to work with Quickstart you have to use `ytt` conform comments (ie. `#!`)